mycelo/genesis/genesis_state.go

```diff
@@ -580,16 +580,16 @@ func (ctx *deployContext) deployValidators() error {
 		return contract.SimpleCall("initialize",
 			env.MustProxyAddressFor("Registry"),
 			ctx.genesisConfig.Validators.GroupLockedGoldRequirements.Value,
-			ctx.genesisConfig.Validators.GroupLockedGoldRequirements.Duration,
+			new(big.Int).SetUint64(ctx.genesisConfig.Validators.GroupLockedGoldRequirements.Duration),
 			ctx.genesisConfig.Validators.ValidatorLockedGoldRequirements.Value,
-			ctx.genesisConfig.Validators.ValidatorLockedGoldRequirements.Duration,
-			ctx.genesisConfig.Validators.ValidatorScoreExponent,
+			new(big.Int).SetUint64(ctx.genesisConfig.Validators.ValidatorLockedGoldRequirements.Duration),
+			new(big.Int).SetUint64(ctx.genesisConfig.Validators.ValidatorScoreExponent),
 			ctx.genesisConfig.Validators.ValidatorScoreAdjustmentSpeed.BigInt(),
-			ctx.genesisConfig.Validators.MembershipHistoryLength,
-			ctx.genesisConfig.Validators.SlashingPenaltyResetPeriod,
-			ctx.genesisConfig.Validators.MaxGroupSize,
-			ctx.genesisConfig.Validators.CommissionUpdateDelay,
-			ctx.genesisConfig.Validators.DowntimeGracePeriod,
+			new(big.Int).SetUint64(ctx.genesisConfig.Validators.MembershipHistoryLength),
```

Comment:
nit: tempted to ask for a `bigInt(uint64)` function, to remove bolierplate code :)